tests/data_files/mock_rbac_returns_multiple_types_of_read_permissions.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta": {
+    "count": 5,
+    "limit": 1000,
+    "offset": 0
+  },
+  "links": {
+    "first": "/api/rbac/v1/access/?application=inventory&limit=1000&offset=0",
+    "next": null,
+    "previous": null,
+    "last": "/api/rbac/v1/access/?application=inventory&limit=1000&offset=0"
+  },
+  "data": [
+    {
+      "resourceDefinitions": [],
+      "permission": "ros:*:*"
+    },
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "group.id",
+            "value": [
+              "404c617d-3dc7-4de6-912b-bde1312e1ce5"
+            ],
+            "operation": "in"
+          }
+        }
+      ],
+      "permission": "inventory:groups:read"
+    },
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "group.id",
+            "value": [
+              "155860d5-648c-4529-847a-690cbf198934"
+            ],
+            "operation": "in"
+          }
+        }
+      ],
+      "permission": "inventory:groups:write"
+    },
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "group.id",
+            "value": [
+              "12345678-fe1b-4191-8408-cbadbd47f7a3"
```

Comment:
@upadhyeammit shall we add a comment mentioning `example-group id` for readability?